plugins/modules/azure_rm_aduser_info.py

```diff
@@ -192,6 +204,8 @@ def __init__(self):
         self.all = None
         self.log_path = None
         self.log_mode = None
+        self.given_name = None
```

Comment:
Not need
```suggestion
```

Reply:
- name: Get Aduser info and filter by givenName
      azure_rm_aduser_info:
        attribute_name: givenName
        attribute_value: Yan